frontend/src/pages/footer.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import {
+    Grid,
+    Typography
+} from '@material-ui/core';
+import { makeStyles } from '@material-ui/core/styles';
+
+const useStyles = makeStyles(theme => ({
+    root: {
+      flexGrow: 1,
+      backgroundColor: '#e0e0e0',
```

Comment:
Can I get this from theme somewhere?

Reply:
Maybe try here? https://material-ui.com/system/palette/#api Not sure 🤔

Reply:
Actually @IVIURRAY I think this is what you want: https://material-ui.com/customization/color/#color-palette

Reply:
```js
import grey from '@material-ui/core/colors/grey';

// ...
const useStyles = makeStyles(theme => ({
    root: {
      flexGrow: 1,
      backgroundColor: grey[300],
// ...
```